package.json

```diff
@@ -5,17 +5,22 @@
   "scripts": {
     "build": "rollup -c",
     "dev": "rollup -c -w",
-    "start": "sirv public --no-clear"
+    "prettier": "prettier --write --plugin-search-dir=. \"{public,src}/**/*.{css,html,svelte,js,mjs,ts}\"",
+    "start": "sirv public --no-clear",
+    "svelte-check": "svelte-check --output human-verbose"
   },
   "devDependencies": {
     "@rollup/plugin-commonjs": "^17.0.0",
     "@rollup/plugin-node-resolve": "^11.0.0",
+    "prettier": "^2.3.1",
+    "prettier-plugin-svelte": "^2.3.1",
```

Comment:
These are just to gussy up the formatting of your code. The `.prettierrc` file above has some common rules I usually use.

Reply:
I mean, I guess this is alright then